yang/iproute2-ip-route.yang

```diff
@@ -58,25 +89,35 @@ module iproute2-ip-route {
             }
             leaf metric {
                 onmcli:key-default-val "0";
+                ipr2cgen:oper-default-val "0";
                 type uint32;
                 description
                     "route metric [default 0]";
             }
             leaf tos {
                 onmcli:key-default-val "default";
+                ipr2cgen:oper-default-val "default";
                 type ipr2net:tos;
                 description
                     "the Type Of Service (TOS) key. This key has no associated mask and the longest match is
                     understood as: First, compare the TOS of the route and of the packet. If they are not equal,
                     then the packet may still match a route with a zero TOS. TOS is either an 8 bit hexadecimal
                     number or an identifier from /etc/iproute2/rt_dsfield.";
             }
+            leaf src {
```

Comment:
src is part of the enca:
```
ENCAP_IP := ip id TUNNEL_ID dst REMOTE_IP [ src SRC ] [ tos TOS ]
               [ ttl TTL ]
```

Reply:
preferred source can be set to a route:
`$ sudo ip route add 10.10.10.0/24 src 10.0.2.15 protocol ra scope site dev enp0s3
`